resharper/resharper-unity/src/CSharp/Feature/Services/Refactorings/Rename/UnityRenameUtil.cs

```diff
@@ -0,0 +1,23 @@
+using JetBrains.ProjectModel;
+using JetBrains.ReSharper.Psi;
+
+namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.Refactorings.Rename
+{
+    public static class UnityRenameUtil
+    {
+        public static bool IsRenameShouldBeSilent(IDeclaredElement declaredElement)
+        {
+            var project = declaredElement.GetSourceFiles().FirstOrDefault()?.GetProject();
+            if (project == null)
+                return false;
+
+            if (project.Name.EndsWith(".Player"))
```

Comment:
Should this be using the new `IsPlayerProject` extension method?

Reply:
Of course, thanks